datafusion/core/src/physical_optimizer/pipeline_checker.rs

```diff
@@ -301,7 +301,7 @@ mod sql_tests {
         let case = QueryCase {
             sql: "SELECT
                     c9,
-                    SUM(c9) OVER(PARTITION BY c1 ORDER BY c9 ASC ROWS BETWEEN 1 PRECEDING AND 5 FOLLOWING) as sum1
+                    SUM(c9) OVER(PARTITION BY c1 ORDER BY c9 ASC ROWS BETWEEN 1 PRECEDING AND UNBOUNDED FOLLOWING) as sum1
```

Comment:
Previous version of the tests were running with `WindowAggExec`(which breaks pipeline)  before this PR. With this PR they run with `BoundedWindowAggExec` (which is pipeline friendly).  Test checks whether `WindowAggExec` breaks pipeline. Hence existing test should be changed to choose `WindowAggExec` instead of `BoundedWindowAggExec`.